Frontend/src/App.jsx

```diff
@@ -1,64 +1,162 @@
-    { id: 2, name: "Dr. Johnson", specialty: "Pediatrics", patients: 200, appointments: 600, experience: 10, qualifications: "MD, FAAP", image: "https://randomuser.me/api/portraits/women/2.jpg", bio: "Dr. Johnson is a compassionate pediatrician dedicated to providing comprehensive care for children of all ages." },
-    { id: 3, name: "Dr. Williams", specialty: "Orthopedics", patients: 150, appointments: 500, experience: 12, qualifications: "MD, FAAOS", image: "https://randomuser.me/api/portraits/men/3.jpg", bio: "Dr. Williams specializes in sports medicine and joint replacement surgeries, helping patients regain mobility and improve their quality of life." },
-    { id: 4, name: "Dr. Brown", specialty: "Neurology", patients: 100, appointments: 350, experience: 18, qualifications: "MD, PhD", image: "https://randomuser.me/api/portraits/women/4.jpg", bio: "Dr. Brown is a leading neurologist with expertise in treating a wide range of neurological disorders and conducting groundbreaking research." },
-    { id: 5, name: "Dr. Taylor", specialty: "Dermatology", patients: 180, appointments: 550, experience: 8, qualifications: "MD, FAAD", image: "https://randomuser.me/api/portraits/men/5.jpg", bio: "Dr. Taylor is a skilled dermatologist specializing in both medical and cosmetic dermatology, helping patients achieve healthy and beautiful skin." },
-    { id: 6, name: "Dr. Alex", specialty: "Gynecologist", patients: 350, appointments: 650, experience: 10, qualifications: "MD, PhD", image: "https://randomuser.me/api/portraits/men/94.jpg", bio: "Dr. Alex is an experienced gynecologist providing comprehensive women's health care, specializing in reproductive health, pregnancy care, and preventive medicine for women of all ages."},
-    { id: 7, name: "Dr. Wilson", specialty: "General practitioner", patients: 250, appointments: 620, experience: 12, qualifications: "MD, PHD", image: "https://randomuser.me/api/portraits/women/6.jpg", bio: "Dr. Wilson is an experienced general practitioner providing comprehensive primary care and preventive medicine for patients of all ages." },
-    { id: 8, name: "Dennis Schmidt", specialty: "Anesthesiologist", patients: 350, appointments: 750, experience: 17, qualifications: "MD, PHD", image: "https://randomuser.me/api/portraits/men/26.jpg", bio: "Dr. Dennis Schmidt is an experienced general practitioner providing comprehensive primary care and preventive medicine for patients of all ages." },
-    { id: 9, name: "Reginald Bennett", specialty: "Ophthalmologist", patients: 150, appointments: 450, experience: 15, qualifications: "MD, PHD", image: "https://randomuser.me/api/portraits/men/18.jpg", bio: "Dr. Reginald Bennett is an experienced general practitioner providing comprehensive primary care and preventive medicine for patients of all ages." },
+    {
+      id: 1,
```

Comment:
Please use Prettier for code formatting to ensure consistency and readability

Reply:
what is the problem I am not getting what I need to do as the code is formatted perfectly from my end and this formatted with the prettier